extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -32,9 +32,19 @@ static public function onGetRailModuleList( &$modules ) {
 		return true;
 	}
 
+	/**
+	 * Hook: BeforePageDisplay
+	 * Add Recirculation Sass styles and JSMessages package to output
+	 * @param OutputPage $out
+	 * @param Skin $skin
+	 * @return bool true to continue hook processing
+	 */
 	public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$out, $skin". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)